source/common/common/backoff_strategy.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/backoff_strategy.h"
+
+namespace Envoy {
+
+ExponentialBackOffStrategy::ExponentialBackOffStrategy(const uint64_t initial_interval,
```

Comment:
We generally don't make scalar args consts in Envoy (although it's not a terrible idea)..